### PR TITLE
[SYCLomatic] Support migration of cublasGetStatusString

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Review by tomflinda — APPROVED** [review]
LGTM

File: clang/lib/DPCT/ASTTraversal.cpp [hunk 1]
```diff
@@ -4643,11 +4643,11 @@ void BLASFunctionCallRule::runRule(const MatchFinder::MatchResult &Result) {
   // TODO: Need to process the situation when scalar pointers (alpha, beta)
   // are device pointers.
 
-  if (MapNames::BLASComputingAPIWithRewriter.find(FuncName) !=
-      MapNames::BLASComputingAPIWithRewriter.end()) {
+  if (MapNames::BLASAPIWithRewriter.find(FuncName) !=
```
Comment (zhiweij1) [inline clang/lib/DPCT/ASTTraversal.cpp]: remove the 2nd time find

Comment (zhiweij1) [reply clang/lib/DPCT/ASTTraversal.cpp]: done

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**